tasks/restic.yml

```diff
@@ -1,34 +1,69 @@
 ---
 # restic tasks file for autorestic
+- name: Ensure restic directory exist
+  ansible.builtin.file:
+    state: directory
+    path: "{{ autorestic_restic_install_dir }}/bin"
```

Comment:
Can we remove the `/bin` from here, and have the absolute path in `autorestic_restic_install_dir`?